internal/kubehelper/client.go

```diff
@@ -11,8 +12,19 @@ import (
 
 func createKubeClient(timeout int64) (*kubernetes.Clientset, error) {
 	var cliKubeconfig string
-	flag.StringVar(&cliKubeconfig, "kubeconfig", "", "absolute path to the kubeconfig file")
-	flag.Parse()
+	for i := 1; i < len(os.Args)-1; i++ {
```

Comment:
Will this catch `--kubeconfig=foo` being the last argument? I assume that `kubectl get pods --kubeconfig=/dev/null` is a valid command?

Was actually thinking about the `cliKubeconfig = os.Args[i+1]` two lines down, but that's safe because of this…

Reply:
> I assume that `kubectl get pods --kubeconfig=/dev/null` is a valid command?

I thought the global options have to be specified before the command verb/word, but I guess not.

Fixed, I think (famous last words alert!).